x-pack/legacy/plugins/apm/cypress/cypress/integration/apm.feature

```diff
@@ -0,0 +1,9 @@
+Feature: APM
+
+  Background:
```

Comment:
Backgrounds are executed before any scenario within this feature file (like a before test method)